runtime-modules/membership/src/lib.rs

```diff
@@ -275,7 +285,14 @@ decl_module! {
             Self::ensure_unique_handle(&user_info.handle)?;
```

Comment:
Do we need this check after copying inside the `insert_member`?

Reply:
quite right, I forgot to remove it after adding the check in `insert_member`

Reply:
done in 2957b71afda80327fb68f9de2686704979bec852